plugins/module_utils/redfish_utils.py

```diff
@@ -2929,10 +2937,9 @@ def virtual_media_eject_via_patch(self, uri, image_only=False):
         return resp
 
     def virtual_media_eject(self, options, resource_type='Manager'):
-        image_url = options.get('image_url')
-        if not image_url:
-            return {'ret': False,
-                    'msg': "image_url option required for VirtualMediaEject"}
+        image_url = None
+        if options:
```

Comment:
Why make options itself optional? Why not simply keep the `image_url = options.get('image_url')`?

Reply:
Previously `image_url` was the only required parameter for `virtual_media`. Since it's optional now a user can exclude the `virtual_media` parameter entirely, which can make `options.get` fail. 

Before:
```yaml
- name: Eject Virtual Media
  community.general.redfish_command:
    category: Systems
    command: VirtualMediaEject
    baseuri: "{{ baseuri }}"
    username: "{{ username }}"
    password: "{{ password }}"
    virtual_media:
      image_url: 'http://example.com/images/SomeLinux-current.iso'
    resource_id: 1
```

after
```yaml
- name: Eject Virtual Media
  community.general.redfish_command:
    category: Systems
    command: VirtualMediaEject
    baseuri: "{{ baseuri }}"
    username: "{{ username }}"
    password: "{{ password }}"
    resource_id: 1
```
It could be changed to `options.get('image_url', None)` if you'd prefer.

Reply:
That indicates to me that the caller already had a bug by passing in `None` when `virtual_media` wasn't provided, which should result in a proper error and not a crash due to `options` being `None`.